task-management/src/main/java/org/exoplatform/task/management/controller/UserController.java

```diff
@@ -68,15 +70,19 @@
     @Inject
     private SpaceService sService;
 
+    @Inject
+    private ProjectService projectService;
+
     @Resource
     @Ajax
     @MimeType.JSON
-    public Response findUser(String query, String projectName) throws Exception { // NOSONAR
+    public Response findUser(String query, String projectName, Long projectId) throws Exception { // NOSONAR
       ListAccess<org.exoplatform.task.model.User> list = userService.findUserByName(query);
       JSONArray array = new JSONArray();
       for(org.exoplatform.task.model.User u : list.load(0, UserUtil.SEARCH_LIMIT)) {
           JSONObject json = new JSONObject();
           Space space = sService.getSpaceByPrettyName(projectName);
+          Space projectSpace = getProjectSpace(projectId);
```

Comment:
```suggestion
          Space spaceProject = getProjectSpace(projectId);
```
Put these instructions:
Space space = sService.getSpaceByPrettyName(projectName);
Space spaceProject = getSpaceProject(projectId);
before for loop